README.md

```diff
@@ -62,7 +62,7 @@ How to get help and news
 
 * Keep up to date on announcements and more by following me [@l0rdn1kk0n](http://twitter.com/l0rdn1kk0n) on Twitter or use the [#WicketBootstrap](https://twitter.com/search?q=%23WicketBootstrap&src=typd) hashtag
 * use the [google group](https://groups.google.com/d/forum/wicket-bootstrap) (wicket-bootstrap (at) googlegroups.com)
-* you can see wicket-bootstrap in action: https://wb8.teliclab.info
+* you can see wicket-bootstrap in action: http://wb.agilecoders.de/
```

Comment:
Let's just remove this.
http://wb.agilecoders.de/ now forwards to https://doridina.de/

Reply:
@martin-g you too fast :)
Shall this be removed in separate PR? :)

Reply:
I decided to do it myself and not bother you :-)